### PR TITLE
Update links in a PR template to avoid 404

### PR DESCRIPTION
**Pull request by ihcomega56**
## Description

Changed links in a PR template as some documents are moved.

Fixes pyrsia/pyrsia#

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### CODE REVIEW

**Review by betarelease — DISMISSED** [review]
👍🏽

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ This PR does... by accomplishing... and it can be reviewed by... you can also te
 This section applies to code modifications, you may remove it otherwise.
 
 Make sure your Pull Request will pass the CI/CD pipeline.
-For a complete list of steps, check out the [developer workflow](https://github.com/pyrsia/pyrsia/blob/main/docs/get_involved/dev_workflow.md)!
+For a complete list of steps, check out the [developer workflow](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/submit_pr.md)!
```
Comment (dasmanas) [inline .github/pull_request_template.md:39]:
```suggestion
For a complete list of steps, check out the [submit pr](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/submit_pr.md)!
```

Comment (ihcomega56) [reply .github/pull_request_template.md:39]: Thank you @dasmanas ! To make the whole sentence easier to understand, how about this?:
```suggestion
For a complete list of steps, check out the [developer PR guidlines](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/submit_pr.md)!
```

Comment (AbhijithGanesh) [reply .github/pull_request_template.md:39]: @ihcomega56 Your suggestion is better 👍

**Review by AbhijithGanesh — APPROVED** [review]
LGTM

**Review by dasmanas — DISMISSED** [review]
lgtm

**Review by dasmanas — APPROVED** [review]

**Review by erwin1 — APPROVED** [review]

**Review by tiainen — APPROVED** [review]

**Pull request merged**